src/SFA.DAS.RequestApprenticeTraining.Domain/Interfaces/IEmployerRequestEntityContext.cs

```diff
@@ -93,7 +135,7 @@ public async Task<List<SelectEmployerRequest>> GetForProviderStandard(long ukprn
                     StandardReference = er.StandardReference,
                     StandardTitle = er.Standard.StandardTitle,
                     StandardLevel = er.Standard.StandardLevel,
-                    DateOfRequest = er.RequestedAt,
+                    DateOfRequest = DateTime.Now,
```

Comment:
I think this should be er.RequestedAt , looks like a merge issue

Reply:
Spot! - Fixed